src/03_modules.py

```diff
@@ -9,23 +9,29 @@
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
```

Comment:
Awesome job getting into the habit of formatting your output, and nice use of enumerate!